sqllogictest/src/runner.rs

```diff
@@ -528,6 +535,13 @@ pub struct Runner<D: AsyncDB, M: MakeConnection> {
     labels: HashSet<String>,
 }
 
+impl<D: AsyncDB, M: MakeConnection<Conn = D>> Drop for Runner<D, M> {
+    fn drop(&mut self) {
+        tracing::debug!("shutting down runner...");
+        block_on(self.conn.shutdown_all());
+    }
+}
```

Comment:
Would `block_on` in `drop` be problematic? (If not, why don't we simply do it in `impl Drop for Postgres` without introducing a separate `shutdown`?

Reply:
This is a good point... 😕

Reply:
Updated to always let the caller handle `shutdown`.